apps/demo-game/src/pages/_app.tsx

```diff
@@ -3,6 +3,7 @@ import { config } from '@fortawesome/fontawesome-svg-core'
 import { useApollo } from '@gbl-uzh/platform/dist/lib/apollo'
 import { SessionProvider } from 'next-auth/react'
 import type { AppProps } from 'next/app'
+import { Toaster } from 'react-hot-toast'
```

Comment:
better: use toaster and toast from shadcn which is available in the design system -> https://ui.shadcn.com/docs/components/toast

Reply:
I have added it through shadcn and created the specific components for that, because it did not work with the design system, yet. We can replace and remove the components as soon as the design system supports the toast functionality.
Personally, I liked the react-hot-toast better, but since we use shadcn everywhere else it makes sense to be consistent.